cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -307,13 +373,15 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             this.parentElement = this.options.parentElement;
             this.render();
         },
-
         render: function() {
             var xblockInfo = this.model;
             var isTimeLimited = xblockInfo.get('is_time_limited');
             var isProctoredExam = xblockInfo.get('is_proctored_exam');
             var isPracticeExam = xblockInfo.get('is_practice_exam');
             var isOnboardingExam = xblockInfo.get('is_onboarding_exam');
+            var enableHideFromTOCUI = xblockInfo.get('enable_hide_from_toc_ui');
```

Comment:
This would be better with `isEnableHideFromTOCUI` but I understand you will have to change  `enable_hide_from_toc_ui` to 'is_enable_hide_from_toc_ui` so is just a minor suggestion

Reply:
Yes, this field was included in the dependency PR. I'll leave the name as it is so as not to affect any other PR related to the functionality.